src/worker.ts

```diff
@@ -1,45 +1,21 @@
-import {
-	addMissedUpdatesRole,
-	callDiscordNicknameBatchUpdate,
-	syncExternalAccounts,
-	syncIdle7dUsers,
-	syncIdleUsers,
-	syncOnboarding31dPlusUsers,
-	syncUnverifiedUsers,
-	syncUsersStatus,
-} from './handlers/scheduledEventHandler';
+import { addMissedUpdatesRoleHandler, callDiscordNicknameBatchUpdateHandler, syncApiHandler } from './handlers/scheduledEventHandler';
 import { env } from './types/global.types';
 
-const EVERY_6_HOURS = '0 */6 * * *';
 const EVERY_11_HOURS = '0 */11 * * *';
 const EVERY_20_MINUTES = '*/20 * * * *';
-const EVERY_30_MINUTES = '*/30 * * * *';
 
 export default {
 	// eslint-disable-next-line no-unused-vars
 	async scheduled(req: ScheduledController, env: env, ctx: ExecutionContext) {
 		switch (req.cron) {
-			case EVERY_6_HOURS: {
-				return await callDiscordNicknameBatchUpdate(env);
-			}
 			case EVERY_11_HOURS: {
-				return await addMissedUpdatesRole(env);
-			}
-
-			case EVERY_20_MINUTES: {
-				await syncIdleUsers(env);
-				// await syncNickNames(env); TODO: Enable it once changes from website-backend is merged
-				await syncIdle7dUsers(env);
-				await syncOnboarding31dPlusUsers(env);
-				console.log('Worker for syncing idle users, nicknames, idle 7d users, and onboarding 31d+ users has completed.');
+				await callDiscordNicknameBatchUpdateHandler(env);
+				await addMissedUpdatesRoleHandler(env);
```

Comment:
So since we are using await here, lets say addMissingUpdatesRoleHandler, runs for more than 10ms then won't the worker stop, and the other call never happen?